node/src/neighborhood/overall_connection_status.rs

```diff
@@ -90,6 +99,13 @@ impl ConnectionProgress {
             )
         };
 
+        debug!(
+            logger,
+            "The node with IP address {:?} has passed to a new node with IP address {:?}.",
```

Comment:
This message should have something in it to make it clear that it has to do with Gossip. Note: the name of the Logger may actually take care of this; I don't know what it is.